jedi/api/classes.py

```diff
@@ -445,9 +455,18 @@ def name_with_symbols(self):
         Similar to :attr:`name`, but like :attr:`name`
         returns also the symbols, for example::
 
-            list()
+            list()| <-- cursor is here
+
+        Would return ``.append`` and others (which means it adds a dot).
```

Comment:
This feature was actually removed a while ago to simplify and generalize completions. I think you can safely remove the above 3 lines.